.github/workflows/pythonpackage.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
-        django-version: ['<3', '>=3']
+        python-version: [3.8]
```

Comment:
Should add 3.9 and 3.10 (should be out, listed on [here](https://en.m.wikipedia.org/wiki/History_of_Python)at least)

Reply:
Looks like `3.10` isn't offered by `actions/setup-python` yet, but I added `3.9`.